src/js/components/modals/ActionsModal.js

```diff
@@ -108,18 +108,20 @@ class ActionsModal extends mixin(StoreMixin) {
       selectedItemsString = selectedItems[0].description;
     } else {
       // Truncate list of selected user/groups for ease of reading
-      let selectedItemsShown = _.first(selectedItems, ITEMS_DISPLAYED + 1);
+      let selectedItemsShown = selectedItems.slice(0, ITEMS_DISPLAYED);
```

Comment:
are you sure you don't need the `+1` anymore? i'm doing it in the console, and first/slice returns the same number of items if the 2nd arg is the same